tests/integration/insert-trailer-registration.int.test.ts

```diff
@@ -0,0 +1,41 @@
+import supertest from 'supertest';
+import { emptyDatabase, populateDatabase } from '../utils/dbUtil';
+import * as domain from '../../src/domain';
+
+const url = 'http://localhost:3020/local';
+const request = supertest(url);
+
+describe('insert trailer registration', () => {
+  beforeAll(async () => {
+    await emptyDatabase();
+  });
+
+  beforeEach(async () => {
+    await populateDatabase();
+  });
+
+  afterEach(async () => {
+    await emptyDatabase();
+  });
+
+  afterAll(async () => {
+    await populateDatabase();
+  });
+
+  test('Saving 17 digit vin', async () => {
```

Comment:
A single test doesn't feel right here.  We could check that:

17 digit vins are progressed OK
16 & 18 digit VINs are OK
Requests with missing or blank/empty strings for mandatory values are processed OK.
If we process a value twice, that we update our Data correctly.

Reply:
Following tests have been added:
- 17 digits test added
- less than 17 added
- missing field added
- existing vin added